OMCompiler/SimulationRuntime/c/simulation/solver/ida_solver.c

```diff
@@ -421,6 +421,16 @@ ida_solver_initial(DATA* data, threadData_t *threadData, SOLVER_INFO* solverInfo
       warningStreamPrint(LOG_STDOUT, 1, "Symbolic Jacobians are currently not supported by DAE Mode. Switch back to numerical Jacobian!");
       idaData->jacobianMethod = COLOREDNUMJAC;
     }
+    if(idaData->jacobianMethod == SYMJAC)
+    {
+      warningStreamPrint(LOG_STDOUT, 1, "Symbolic Jacobians without coloring are currently not supported by IDA. Colored symbolical Jacobian will be used.");
+      idaData->jacobianMethod = COLOREDSYMJAC;
+    }
+    if(idaData->jacobianMethod == NUMJAC)
+        {
```

Comment:
@AnHeuermann  Nitpicking, but try to follow the same indention as the other things in the file : )